### PR TITLE
chore: fix dev tests

### PR DESCRIPTION
**Pull request by sobolk**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-e2e-core/src/configure/index.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export const amplifyRegions = [
   'us-west-1',
   'us-west-2',
   'eu-north-1',
+  'eu-south-1',
```
Comment (sobolk) [inline packages/amplify-e2e-core/src/configure/index.ts:23]: https://github.com/aws-amplify/amplify-cli/blob/137b629c9e74e4ab8ec90f4519f947ed748f47cf/packages/amplify-provider-awscloudformation/src/aws-regions.js#L1-L4

File: packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts [hunk 2]
```diff
@@ -15,13 +15,15 @@ const AWS_REGIONS_TO_RUN_TESTS = [
   'us-east-2',
   'us-west-2',
   'eu-west-2',
+  'eu-west-3',
   'eu-central-1',
   'ap-northeast-1',
+  'ap-northeast-2',
```
Comment (sobolk) [inline packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts:21]: these new regions were showing up in error messages. we map to them sometimes.

File: packages/amplify-cli/package.json [hunk 3]
```diff
@@ -47,6 +47,10 @@
     "@aws-amplify/amplify-category-predictions": "5.0.1",
     "@aws-amplify/amplify-category-storage": "3.7.4",
     "@aws-amplify/amplify-cli-logger": "1.3.0",
+    "@aws-amplify/amplify-cli-shared-interfaces": "1.1.2",
+    "@aws-amplify/amplify-console-hosting": "2.4.4",
+    "@aws-amplify/amplify-container-hosting": "2.5.14",
+    "@aws-amplify/amplify-dotnet-function-template-provider": "2.4.6",
```
Comment (sobolk) [inline packages/amplify-cli/package.json:53]: This got lost on recent release merge.
image

File: packages/amplify-e2e-core/src/configure/index.ts (hunk 1 above)
Comment (evcodes) [reply packages/amplify-e2e-core/src/configure/index.ts:23]: helpful comment 😛

File: packages/amplify-cli/package.json (hunk 3 above)
Comment (evcodes) [reply packages/amplify-cli/package.json:53]: 😢 Thanks for bringing this back

**Review by lazpavel — APPROVED** [review]

**Review by evcodes — APPROVED** [review]

**Review by sdstolworthy — APPROVED** [review]

**Pull request merged**